.github/workflows/ci.yml

```diff
@@ -102,7 +82,7 @@ jobs:
           # version of everything. We use cabal because it uses the latest
           # versions of our dependencies allowed by our upper bounds.
           - name: newest
-            ghc: 8.4.4
+            ghc: 9.0.1
```

Comment:
I know I told you to use `9.0.1` instead of `latest` in order to work around a bug in haskell/actions/setup, but I just tried it with `ghc: latest` in `hint` and it does use ghc09.0.1: https://github.com/haskell-hint/hint/pull/117/checks?check_run_id=2102650915

so I think we should use `ghc: latest` after all

Reply:
turns out I was misremembering, it's not `ghc: latest` which is buggy, it's `cabal: latest`. And it's you who [reported that bug](https://github.com/haskell/actions/issues/28)!

Reply:
looks like that bug has since been resolved, actually? in the CI logs above, I'm using `cabal: latest` (well, technically I'm using `cabal: <nothing>`, which is equivalent) and it's cabal-3.4 which is being installed this time, not cabal-3.2.

Reply:
> And it's you who [reported that bug](https://github.com/haskell/actions/issues/28)!

Yes I did because at the time cabal-install-3.4 was still an RC